ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -614,6 +864,49 @@ public void replicaTokenTest() throws InterruptedException {
     return ids;
   }
 
+  private BlobId generateRandomBlobId(PartitionId partitionId) {
+    short accountId = Utils.getRandomShort(TestUtils.RANDOM);
+    short containerId = Utils.getRandomShort(TestUtils.RANDOM);
+    short blobIdVersion = CommonTestUtils.getCurrentBlobIdVersion();
+    boolean toEncrypt = Utils.getRandomShort(TestUtils.RANDOM) % 2 == 0;
+    BlobId blobId =
+        new BlobId(blobIdVersion, BlobId.BlobIdType.NATIVE, ClusterMapUtils.UNKNOWN_DATACENTER_ID, accountId,
+            containerId, partitionId, toEncrypt, BlobId.BlobDataType.DATACHUNK);
+    return blobId;
+  }
+
+  private void addPutMessagesToReplicasOfPartition(List<StoreKey> ids, List<Host> hosts)
+      throws MessageFormatException, IOException {
+    List<Transformer> transformerList = new ArrayList<>();
+    for (int i = 0; i < ids.size(); i++) {
+      transformerList.add(null);
+    }
+    addPutMessagesToReplicasOfPartition(ids, transformerList, hosts);
+  }
+
+  private void addPutMessagesToReplicasOfPartition(List<StoreKey> ids, List<Transformer> transformPerId,
```

Comment:
same here - `transformPerId` is not an obvious argument that I can understand from the name.

It's probably ok to skip docs for really simple functions (although I would say that simple functions can become complex later and it is best to add docs to at least signal the intent and encourage doc addition if additional parameters are added, many a time, it looks like we skip adding docs if there isn't already something) but we _have_ to add docs if the behavior/arg is not glaringly obvious. Two main reasons are that I don't have to necessarily read the whole function if I'm just trying to understand the test and second, that I know your intent when reviewing the function.